.env.dist

```
@@ -116,9 +116,9 @@ RALPH_BACKENDS__HTTP__LRS__STATEMENTS_ENDPOINT=/xAPI/statements
 
 # RALPH_CONVERTER_EDX_XAPI_UUID_NAMESPACE=
 
-# LRS API
+# LRS API 
 
-RALPH_RUNSERVER_AUTH_BACKEND=basic
+RALPH_RUNSERVER_AUTH_BACKENDS=Basic
```

If we choose enum values to be lower case:

```suggestion
RALPH_RUNSERVER_AUTH_BACKENDS=basic
```

---

changed this